src/components/BookList.js

```diff
@@ -0,0 +1,18 @@
+import NavBar from './NavBar';
+import Book from './Book';
+import AddBook from './AddBook';
+
+const BookList = () => (
+  <div>
+    <NavBar />
+    <div>
+      <Book title="book 1" author="author 1" />
+      <Book title="book 1" author="author 1" />
+      <Book title="book 1" author="author 1" />
+      <Book title="book 1" author="author 1" />
+    </div>
+    <AddBook />
```

Comment:
- [ ] kindly remove this as it is an empty component causing your app not to run